code/__DEFINES/dcs/signals.dm

```diff
@@ -284,10 +284,14 @@
 
 // /obj/item signals for economy
 #define COMSIG_ITEM_SOLD "item_sold"							//called when an item is sold by the exports subsystem
+#define COMSIG_ITEM_SOLD_MATERIAL "item_sold_material"			//called when an item is sold by the exports subsystem using material datums.
 #define COMSIG_STRUCTURE_UNWRAPPED "structure_unwrapped"		//called when a wrapped up structure is opened by hand
 #define COMSIG_ITEM_UNWRAPPED "item_unwrapped"					//called when a wrapped up item is opened by hand
 	#define COMSIG_ITEM_SPLIT_VALUE  1
+	#define COMSIG_ITEM_SPLIT_VALUE_MATERIAL  1
```

Comment:
No need for two such definitions.
Prefer to use `(1<<0)` instead of `1` as the return of signals are always flags.